config/initializers/external_proof_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ExternalProofService
+  def self.my_domain
+    'mastodon.social'
+  end
+
+  def self.my_domain_displayed
+    my_domain.gsub('.', ' ').capitalize
+  end
+
+  module Keybase
+    def self.base_url
+      'https://keybase.io'
+    end
+
+    def self.my_contacts
+      ["admin@#{ExternalProofService.my_domain}", "yournamehere@keybase"]
```

Comment:
This would probably end up a `Setting` value just like `Setting.bootstrap_timeline_accounts`

Reply:
oooo i like this.

Reply:
i haven't seen this pattern for settings, but i like it.